task-service/src/main/java/com/infodation/task_service/services/TaskServiceImpl.java

```diff
@@ -36,6 +33,18 @@ public class TaskServiceImpl implements ITaskService {
     private  final TaskServiceRepository taskServiceRepository;
     private static final Logger log = LoggerFactory.getLogger(TaskServiceImpl.class);
 
+    SimpleDateFormat dueDateFormatter = new SimpleDateFormat("yyyy-MM-dd");
+    SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
It feels like we use these `SimpleDateFormat` in multiple places.
I suggest creating a Bean for these two `SimpleDateFormat` and do inject into classes that we need